### DISCUSSION TITLE
CLOSED:  Anomaly withBeats and Measures Timeline versus Timer and Delection toolbars

### DISCUSSION THREAD

**Discussion by petersampsonaudacity** [audacity/audacity#4472] [Near future]
Is there a good design/UX reason that the new **Beats and Measures** timeline view start at 1 rather than zero?

This is in stark contrast to the default **Minutes and Seconds** timeline that starts at T=0

In particular this makes the Timeline and the **Time** toolbar (and **Selection** toolbar) look out of step with each other:

![image](https://user-images.githubusercontent.com/50205984/228792042-52ff5ef5-cfaa-488f-9f5d-0174c344a962.png)

Flagging @Tantacrul @DilsonsPickles and @LWinterberg as a potential Design/UX issue

**Comment by crsib** [comment]
Musical people are not used to zero based enumeration :-) It is common that bars and beats are starting with 1, not with 0.

![image](https://user-images.githubusercontent.com/2660628/228799552-cd30be43-124e-49ad-bd13-252da1ffad07.png)

**Comment by petersampsonaudacity** [comment]
Ah thanks Dmitry  

_Reminds me of Fortran programmers who start their arrays counting at 1 - unlike Algol programmer who start arrays at 0_

**Comment by petersampsonaudacity** [comment]
The former discrepancy is now fixed in 3.3.0 Beta

Time toolbar and Selection toolbar are now properly **_in-step_** with the Timeline

![image](https://user-images.githubusercontent.com/50205984/231427071-54984908-2007-4b0d-b238-d3142aa4d5d8.png)

So for me this is a RESOLVED issue - but I can't see how a "Discussion! can be marked as CLOSED
